src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/include/api_conformance_helpers.hpp

```diff
@@ -17,7 +17,7 @@ inline const std::string get_plugin_lib_name_by_device(const std::string& device
             { "HETERO", "openvino_hetero_plugin" },
             { "BATCH", "openvino_auto_batch_plugin" },
             { "MULTI", "openvino_auto_plugin" },
-            { "VPUX", "openvino_intel_vpux_plugin" },
+            { "VPU", "openvino_intel_vpux_plugin" },
```

Comment:
what is the proper plugin name?

Reply:
We intend to use only `VPU` in the future.

Reply:
At the moment we use a hybrid scenario where both `VPU` and `VPUX` devices are recognized, but `VPU` is a [pseudo plugin](https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/pull/4549) for `VPUX` using the same `openvino_intel_vpux_plugin` library. However, in the future we plan to accept only `VPU` device, so library name should also be changed.